MPF.Frontend/Tools/SubmissionGenerator.cs

```diff
@@ -94,9 +95,12 @@ internal static class SubmissionGenerator
                 info.TracksAndWriteOffsets.ClrMameProData = null;
 
             // Add the volume label to comments, if possible or necessary
-            string? volLabels = FormatVolumeLabels(drive?.VolumeLabel, processor?.VolumeLabels);
-            if (volLabels != null)
-                info.CommonDiscInfo!.CommentsSpecialFields![SiteCode.VolumeLabel] = volLabels;
+            if (system != RedumpSystem.MicrosoftXbox && system != RedumpSystem.MicrosoftXbox360)
```

Comment:
There are a few unique volume labels for XGD1/2/3 discs. `GetRedumpSystemFromVolumeLabel` inside of `FormatVolumeLabels` should catch the others.

Reply:
All Xbox360 volume labels are known and defined exactly by the PFI (and SS wave)

Reply:
It'd be nice to get those into `GetRedumpSystemFromVolumeLabel` then :D

Reply:
You already commented some out :)

Reply:
Oh no, `CD_ROM` and `DVD_ROM`???

Reply:
Fine, dropping this one for now. You win this time `CD_ROM`.